### PR TITLE
Auto deploy gateway dev

### PR DESCRIPTION
**Pull request by tboegner**

### CODE REVIEW

File: infrastructure/.env [hunk 1]
```diff
@@ -1,5 +1,7 @@
 IRIS_VERSION=0.0.1-SNAPSHOT
 IRIS_FRONTEND_VERSION=develop
+GATEWAY_VERSION=develop
+EPS_VERSION=develop
```
Comment (lucky-lusa) [inline infrastructure/.env:4]: Lets use 0.1.32 for a start here. EPS updates are expected to result in changes to the settings and thus to the entire deployment. Will be a manual process.

Comment (tboegner) [reply infrastructure/.env]: Now EPS runs with v0.1.32, Proxy and SD with GATEWAY_VERSION (develop).

Comment (tboegner) [reply infrastructure/.env]: @lucky-lusa : EPS=v0.1.32, Proxy & SD = develop. Is this OK?

Comment (tboegner) [reply infrastructure/.env]: Using v0.1.36 for all services from eps repo now.

**Pull request merged**